optuna_dashboard/_app.py

```diff
@@ -449,6 +452,44 @@ def save_trial_note(study_id: int, trial_id: int) -> dict[str, Any]:
         response.status = 204  # No content
         return {}
 
+    @app.get("/csv/<study_id:int>")
+    def download_csv(study_id: int) -> BottleViewReturn:
+        # Create a CSV file
+        summary = get_study_summary(storage, study_id)
+        if summary is None:
+            response.status = 404  # Not found
+            return {"reason": f"study_id={study_id} is not found"}
+        trials = get_trials(storage, study_id)
+
+        param_names = sorted(set(chain.from_iterable([t.params.keys() for t in trials])))
+        user_attr_names = sorted(set(chain.from_iterable([t.user_attrs.keys() for t in trials])))
+
+        param_names_header = [f"Param {x}" for x in param_names]
+        user_attr_names_header = [f"UserAttribute {x}" for x in user_attr_names]
+        value_header = ["Value"]
+        if len(trials[0].values) > 1:
+            value_header = [f"Objective {x}" for x in range(len(trials[0].values))]
```

Comment:
If all trials are either waiting or running, an error happens.

```suggestion
        n_objectives = max([len(t.values) for t in trials if t.values is not None], default=1)
        if n_objectives > 1:
            value_header = [f"Objective {x}" for x in range(n_objectives)]
```

Reply:
Thank you; sorry for my oversights